docs/develop/09_polarpath/01_overview.md

```diff
@@ -0,0 +1,91 @@
+---
+id: what-is-polarpath
+title: Overview
+---
+
+Polar path consists of an `asset-swap` pallet for KILT chain runtimes and extrinisics that let parachain-based chain developers to make their native token accessible on the Ethereum network using [the Snowbridge bridge](https://docs.snowbridge.network) between Polkadot and Ethereum.
+
+Polar path provides parachains with a solution for creating an ERC-20 wrapper around their native tokens, enabling a trustless conversion of any existing native parachain token (PARA) into a wrapped parachain token (wPARA) on Ethereum while maintaining the trustlessness guarantees Snowbridge provides.
+
+The pallet enables a configured conversion rate of parachain tokens on the origin chain to an ERC-20 wrapper token on Polkadot's AssetHub.
+
+## Setup
+
+:::tip Asset Hub
+The Polkadot Relay Chain doesn't natively support assets apart from DOT. Instead, parachains support this functionality, and for Polkadot, this parachain is the [Asset Hub](https://wiki.polkadot.network/docs/build-integrate-assets).
+:::
+
+Making a Parachain Token to be usable on Ethereum requires the following steps:
+
+1. Deploy an ERC-20 contract token to Ethereum with a fixed supply based on the parachain needs.
+2. [Disable the token issuer rights](https://ethereum.org/en/guides/how-to-revoke-token-access/) on the ERC-20 contract.
+3. [Register the ERC-20 token on Asset Hub](https://wiki.polkadot.network/docs/learn-assets#creation-and-management).
+4. The entity holding ERC-20 funds performs a one-time transfer of all tokens from their Ethereum account to the parachain's sovereign account on Asset Hub.
+5. Parachain governance recognizes the wrapper around the ERC-20 token by pairing the ERC-20 token to the parachain token in the swap pallet and enable the token issuer rights.
+
+Holders of the token on the parachain can can now switch tokens between it and Ethereum.
+
+## Use cases
+
+After setup, the following flows are possible:
+
+:::tip XCM
+Polar path uses Polkadot's [Cross-Consensus Message Format (XCM)](https://wiki.polkadot.network/docs/learn-xcm), a messaging format and language used to communicate between consensus systems.
+:::
+
+### Sending tokens from the parachain to Ethereum
+
+1. Transmit custom parachain tokens, "PARA", from the parachain to wrapped tokens "wPARAs" on Asset Hub via XCM
+2. Send wPARAs from Asset Hub to the ERC-20 contract on Ethereum via Snowbridge
+
+### Sending tokens from Ethereum to the parachain
+
+1. Send wPARAs from the ERC-20 contract on Ethereum to Asset Hub via Snowbridge
+2. Transmit wPARAs on Asset Hub to PARAs on the parachain via XCM
+
+## Extrinsics
+
+Before performing a switch, you first need to create a switch pair between the parachain token and the ERC-20 token using the `setSwitchPair` extrinisc. Then you can use any of the other extriniscs with the switch pair, including performing the switch itself using the `switch` extrinisc.
```

Comment:
It's not really you, right? It's a configurable origin, and in our case it will be a public referendum.

Reply:
I'm a bit confused… Someone has to start the process yes? Someone needs to create the switch pair?

Or are there two parts to this?

Someone creates the switch pair and specifies who else can use the other extrinisics.

And what's the referendum? I've not seen anything mentioned in any of the source material about how any of the pallets refer to a referendum. Do you mean that the referendum triggers the other extrinsic, or something elese?

Reply:
A new switch pair can be created by the origin configured. This is generic. In our case, it will be the `Root` origin, which is only possible to use after a referendum. So yes, someone proposes a referendum that would dispatch a specific function (in this case the switch pair creation and enactment), and if it goes through (after ~7 days), the call is dispatched, and the switch pair is created.